projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -120,17 +121,24 @@ function NewsletterNotice( {
 		reachCount = reachCount + '+'; // Concat "+"
 	}
 
+	/* translators: %s is the number of subscribers in numerical format */
+	let numberOfSubscribersText = __(
```

Comment:
This breaks the build, we'll need to move the translator comment:

https://github.com/Automattic/jetpack/actions/runs/4982394240/jobs/8917992184?pr=30668

Reply:
Interesting!  Any ideas as to where we should move this to or what should be changed?

Reply:
What about being more verbose, like this?

```diff
diff --git a/projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js b/projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js
index c58884fcfa..69c12bf48d 100644
--- a/projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js
+++ b/projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js
@@ -121,21 +121,24 @@ export function NewsletterNotice( {
 		reachCount = reachCount + '+'; // Concat "+"
 	}
 
-	/* translators: %s is the number of subscribers in numerical format */
-	let numberOfSubscribersText = __(
-		'This will be sent to <br/><strong>%s subscribers</strong>.',
-		'jetpack'
+	let numberOfSubscribersText = sprintf(
+		/* translators: %s is the number of subscribers in numerical format */
+		__( 'This will be sent to <br/><strong>%s subscribers</strong>.', 'jetpack' ),
+		reachCount
 	);
 
 	if ( isPostPublishPanel ) {
-		/* translators: %s is the number of subscribers in numerical format */
-		numberOfSubscribersText = __( 'This was sent to <strong>%s subscribers</strong>.', 'jetpack' );
+		numberOfSubscribersText = sprintf(
+			/* translators: %s is the number of subscribers in numerical format */
+			__( 'This was sent to <strong>%s subscribers</strong>.', 'jetpack' ),
+			reachCount
+		);
 	}
 
 	return (
 		<FlexBlock>
 			<Notice status="info" isDismissible={ false } className="edit-post-post-visibility__notice">
-				{ createInterpolateElement( sprintf( numberOfSubscribersText, reachCount ), {
+				{ createInterpolateElement( numberOfSubscribersText, {
 					br: <br />,
 					strong: <strong />,
 				} ) }

```

Reply:
Thanks for the tip! Fixed in [e495bcf](https://github.com/Automattic/jetpack/pull/30668/commits/e495bcf5a91b419f8f6ab645f9bf3a9a501996d8).